src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -457,6 +458,7 @@ static ApplicationInsightsTelemetry()
                         "PSScheduledJob",
                         "PSWorkflow",
                         "PSWorkflowUtility",
+                        "PSScriptAnalyzer",
```

Comment:
Shouldn't this be after line 458 keeping alphabetical order?

Reply:
Thanks, updated both to be in alphabetical order